### ISSUE TITLE
Question on Web Scraper

### ISSUE THREAD

**Issue by maysam19** [tattooday/web-scraping#1]
Hello there I hope all is well.

Thank you so much for:

https://github.com/tattooday/web-scraping/blob/master/CME2.py

Its helping a lot, as I am basically looking for a python scraper to scrape futures prices. Just one question for you.

In your code, you designate code numbers for silver gold palladium and copper to be:

458, 437, 445, and 438

I am curious where I would go to find these sequence matchups? Along with find more alternatives and associated codes so I can scrape them too, such as wheat, oil, gold, etc....

Basically where do I go on the web to see what how silver is linked to 458 for example , and gold is linked to 437. And also find codes for other commodities. 

I really appreciate your help in advance and hope to speak soon!

Best,

Sam

**Comment by tattooday** [comment]
Hi Sam

Thx. I am glad you like it.
Actually I wrote that scraper a long while ago.
There are so many redundant lines.
I'd suggest you to optimize it a lil bit for the sake of better performance.
        
The simple answer to your question is,
https://www.cmegroup.com/CmeWS/mvc/ProductSlate/V2/List
You can use pd.read_json to handle it and find the codes for specific commodities.
The long answer would be right click your web browser.
Inspect the network and check the traffic one by one.
You gotta find out where the traffic is coming from and what it is about.
        
Hopefully it helps.
        
T.M.
